packages/babel-helper-wrap-function/src/index.js

```diff
@@ -12,7 +12,7 @@ const buildWrapper = template(`
   })
 `);
 
-const namedBuildWrapper = template(`
+const buildNamedExpressionWrapper = template.expression(`
```

Comment:
Both the `buildNamedExpressionWrapper` and `buildWrapper` templates are, as far as is observable, identical at runtime, so maybe `buildWrapper` should be the `buildExpressionWrapper` (smaller code size). It even correctly accepts a `NAME` parameter.